tests/testMaskedImage.py

```diff
@@ -120,9 +120,10 @@ def tearDown(self):
 
     def testArrays(self):
         image, mask, variance = self.mimage.getArrays()
-        self.assert_((self.mimage.getImage().getArray() == image).all())
-        self.assert_((self.mimage.getMask().getArray() == mask).all())
-        self.assert_((self.mimage.getVariance().getArray() == variance).all())
+        self.assertListEqual(self.mimage.getImage().getArray().flatten().tolist(), image.flatten().tolist())
```

Comment:
OH! I get it. Please add a docstring to the method explaining that. It makes sense now, but took some thinking to understand, so it should be documented better.